plugins/job-handler/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@flatfile/plugin-job-handler",
+  "version": "0.0.0",
+  "description": "A plugin for handling Flatfile Jobs.",
+  "registryMetadata": {
+    "category": "handlers"
```

Comment:
@ashleygmulligan2 / @bangarang  do you have a preference for a category here?

Reply:
let's do core/